docs/resources/vpc.md

```diff
@@ -47,6 +47,7 @@ resource "ybm_vpc" "example-vpc" {
 ### Read-Only
 
 - `account_id` (String) The ID of the account this VPC belongs to.
+- `external_vpc_id` (String) The VPC ID for the YB Managed VPC in the cloud provider(AWS/GCP/Azure)
```

Comment:
Can we be a bit more specific? If I understand correctly - this is the VPC ID where YBM resources are created?